nms/app/packages/magmalte/grafana/dashboards/CWFDashboards.js

```diff
@@ -100,6 +100,47 @@ export const CWFSubscriberDBData: GrafanaDBData = {
           description:
             "Should just be 0 for subscribers without an active session and 1 for those with. Not sure what's up with this now.",
         },
+      ]
+    },
+    {
+      title: "Aggregated Sessions",
+      panels: [
+        {
+          title: 'User Growth',
+          targets: [
+            {
+              expr: 'count(sum(session_start{msisdn=~"$msisdn"}) by (msisdn))',
```

Comment:
So now because you use `{msisdn="$msisdn"}` these metrics will no longer be aggregated all the time and if you select a single subscriber it will only show stats for that one. I'd say either go back to using `.*` for msisdn or adjust the description/title of the row

Reply:
Accepted and incorporated. Will use .*